utils/wallet/wallet.go

```diff
@@ -94,6 +94,15 @@ func IsNotFound(err error) bool {
 	return ok && te.NotFoundError()
 }
 
+// IsInvalidDestination is a helper method for determining if an error indicates an invalid destination
+func IsInvalidDestination(err error) bool {
```

Comment:
Confusing, maybe "IsInvalidDestinationErr" or "IsErrInvalidDestination" or similar?  

I read this function signature as "is the input parameter an invalid destination?"

Reply:
Yeah we should probably rename all these or move them to a helper module, I'll make a follow up issue

Reply:
https://github.com/brave-intl/bat-go/issues/491